static/locales/en.client.json

```diff
@@ -1536,7 +1537,10 @@
         "Security Settings": "Security Settings",
         "Updates": "Updates",
         "unconfigured": "unconfigured",
-        "unknown": "unknown"
+        "unknown": "unknown",
+        "Authentication": "Authentication",
+        "Current authentication method": "Current authentication method",
+        "Grist allows different types of authentication to be configured, including SAML and OIDC.     We recommend enabling one of these if Grist is accessible over the network or being made available     to multiple people.": "Grist allows different types of authentication to be configured, including SAML and OIDC.     We recommend enabling one of these if Grist is accessible over the network or being made available     to multiple people."
```

Comment:
@Spoffy  the translation key looks iffy here. Maybe the spacing should be changed. What do other long keys in the codebase do?